src/util/debug_module.js

```diff
@@ -517,6 +525,7 @@ DebugLogger.prototype.warn = log_syslog_builder('warn');
 DebugLogger.prototype.info = log_syslog_builder('info');
 DebugLogger.prototype.log = log_syslog_builder('log');
 DebugLogger.prototype.trace = log_syslog_builder('trace');
+DebugLogger.prototype.event = log_syslog_builder('event');
```

Comment:
I think we should avoid using this `log_syslog_builder`, especially considering you later remove all the formatting added by this function. 
I suggest adding a simple function to log the event.

Also, maybe we should add a global toggle to disable event logging, just in case?

```javascript
function log_event(event) {
    if (!config.EVENT_LOGGING_ENABLED) return;// necessary?
    syslog(int_dbg._levels_to_syslog.EVENT, JSON.stringify(event), config.EVENT_FACILITY);
}
```

and export it as the `event` function
```suggestion
DebugLogger.prototype.event = log_event;
```